delivery_roulier_laposte/models/stock.py

```diff
@@ -130,4 +130,8 @@ def _laposte_convert_address(self, partner):
         if 'partner_firstname' in self.env.registry._init_modules \
                 and partner.firstname:
             address['firstName'] = partner.firstname
+        # because only mobile is required
+        # and phone key is used laposte roulier template
+        if address.get('mobile'):
+            address['phone'] = address.get('mobile')
```

Comment:
if false then `address['phone'] = address.get('mobile')` is not executed

if you're ok, I only merge the first commit (ONE) in other PR